target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst

```diff
@@ -0,0 +1,28 @@
+C:\Users\Kasia\MojeProjekty\eclipse-workspace\TicketBookingSystem\src\pl\michaeli\model\ScreeningMapper.java
+C:\Users\Kasia\MojeProjekty\eclipse-workspace\TicketBookingSystem\src\pl\michaeli\model\Reservation.java
+C:\Users\Kasia\MojeProjekty\eclipse-workspace\TicketBookingSystem\src\pl\michaeli\spring\dao\ScreeningsDAO.java
+C:\Users\Kasia\MojeProjekty\eclipse-workspace\TicketBookingSystem\src\pl\michaeli\spring\dao\ReservationsDAO.java
+C:\Users\Kasia\MojeProjekty\eclipse-workspace\TicketBookingSystem\src\pl\michaeli\model\TicketTypeMapper.java
+C:\Users\Kasia\MojeProjekty\eclipse-workspace\TicketBookingSystem\src\pl\michaeli\model\Seat.java
+C:\Users\Kasia\MojeProjekty\eclipse-workspace\TicketBookingSystem\src\pl\michaeli\model\BookedSeat.java
+C:\Users\Kasia\MojeProjekty\eclipse-workspace\TicketBookingSystem\src\pl\michaeli\model\Screening.java
+C:\Users\Kasia\MojeProjekty\eclipse-workspace\TicketBookingSystem\src\pl\michaeli\model\FilmMapper.java
+C:\Users\Kasia\MojeProjekty\eclipse-workspace\TicketBookingSystem\src\pl\michaeli\service\ScreeningsResource.java
+C:\Users\Kasia\MojeProjekty\eclipse-workspace\TicketBookingSystem\src\pl\michaeli\model\SeatMapper.java
+C:\Users\Kasia\MojeProjekty\eclipse-workspace\TicketBookingSystem\src\pl\michaeli\spring\dao\VouchersDAO.java
+C:\Users\Kasia\MojeProjekty\eclipse-workspace\TicketBookingSystem\src\pl\michaeli\service\AvailableSeatsResource.java
+C:\Users\Kasia\MojeProjekty\eclipse-workspace\TicketBookingSystem\src\pl\michaeli\model\BookedSeatMapper.java
+C:\Users\Kasia\MojeProjekty\eclipse-workspace\TicketBookingSystem\src\pl\michaeli\model\ErrorMessage.java
+C:\Users\Kasia\MojeProjekty\eclipse-workspace\TicketBookingSystem\src\pl\michaeli\spring\dao\SeatsDAO.java
+C:\Users\Kasia\MojeProjekty\eclipse-workspace\TicketBookingSystem\src\pl\michaeli\spring\dao\BookedSeatsDAO.java
+C:\Users\Kasia\MojeProjekty\eclipse-workspace\TicketBookingSystem\src\pl\michaeli\model\ReservationMapper.java
+C:\Users\Kasia\MojeProjekty\eclipse-workspace\TicketBookingSystem\src\pl\michaeli\model\Voucher.java
+C:\Users\Kasia\MojeProjekty\eclipse-workspace\TicketBookingSystem\src\pl\michaeli\model\Film.java
```

Comment:
O, ciekawe. Polecam dać szansę Intellij IDEA jako alternatywę dla Eclipse.

Reply:
Właściwie to wolę korzystać z Intellij'a :-)